.frigg.yml

```diff
@@ -2,7 +2,8 @@ setup_tasks:
  - npm install
 
 tasks:
- - npm uninstall react-dom
+ - npm run lint
+ - npm test
+ - npm uninstall react-dom react-addons-test-utils
  - npm install react@0.13 && npm test
- - npm install react@latest react-dom@latest && npm test
  - npm run lint
```

Comment:
@relekang I changed the order of testing here. It makes more sense to test with React 0.14.x first. It removes the need to reinstall react-dom and react-addons-test-utils after testing 0.13.x. It also means we can use the version in package.json instead of 'latest'.